package/utils/omnia-eeprom/Makefile

```
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2024 Marek Behún <kabel@kernel.org>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=omnia-eeprom
+PKG_VERSION:=v0.1
```

I think that your versioning is not compatible with APK.

It should be without the v before the version:
```suggestion
PKG_VERSION:=0.1
```

Unfortunately, you need to tweak other variables.